README.md

```diff
@@ -1,2 +1,71 @@
-# ballerina-openapi
-Ballerina OpenApi-Tool
+# Ballerina-OpenAPI
+ [![Build master](https://github.com/ballerina-platform/ballerina-openapi/workflows/Build%20master/badge.svg?branch=master)](https://github.com/ballerina-platform/ballerina-openapi/actions?query=workflow%3ABuild)
+ [![Daily build](https://github.com/ballerina-platform/ballerina-openapi/workflows/Daily%20build/badge.svg)](https://github.com/ballerina-platform/ballerina-openapi/actions?query=workflow%3A%22Daily+build%22)
+ [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/ballerina-openapi.svg)](https://github.com/ballerina-platform/ballerina-openapi/commits/master)
+ 
+The OpenAPI Specification is a specification, which creates a RESTFUL contract for APIs detailing all of its resources 
+and operations in a human and machine-readable format for easy development, discovery, and integration. Ballerina
+ OpenAPI tooling will make it easy for users to start the development of a service documented in the OpenAPI contract 
+  by generating the Ballerina service and client skeletons. The OpenAPI tools provide the following capabilities.
+ 
+ 1. Generate the Ballerina service or client code for a given OpenAPI definition. 
+ 2. Export the OpenAPI definition of a Ballerina service.
+ 3. Validate service implementation of a given OpenAPI Contract.
+    
+The `openapi` command in Ballerina is used for OpenAPI to Ballerina and Ballerina to OpenAPI code generations. 
+The OpenAPI compiler plugin will allow you to validate a service implementation against an OpenAPI contract during
+ compile time. 
+This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract.
+For more information on the operations supported, you can refer [OpenAPI Learn Page](https://ballerina.io/swan-lake/learn/using-the-openapi-tools/)
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
```

Comment:
```suggestion
1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
```

This section has updates of the JDL 11 migration done via https://github.com/ballerina-platform/ballerina-dev-website/pull/1381. Please change accordingly.